torch/_utils.py

```diff
@@ -513,3 +514,21 @@ def _handle_complex(tensor):
     """
     return torch.view_as_real(tensor) if not isinstance(tensor,
                                                         torch.nn.UninitializedParameter) and tensor.is_complex() else tensor
+
+
+def _element_size(dtype):
+    """
+    Returns the element size for a dtype, in bytes
+    """
+    if not isinstance(dtype, torch.dtype):
+        raise RuntimeError(f'expected torch.dtype, but got {type(dtype)}')
+
+    if dtype.is_complex:
+        return torch.finfo(dtype).bits >> 2
+    elif dtype.is_floating_point:
+        return torch.finfo(dtype).bits >> 3
+    elif dtype == torch.bool:
+        # NOTE: torch.bool is not supported in torch.iinfo()
+        return 1
+    else:
+        return torch.iinfo(dtype).bits >> 3
```

Comment:
Huh, it's weird that we never had this already

Reply:
Yeah I thought so too. It's possible that we do have a similar function elsewhere, but I couldn't find it